src/nidigital/system_tests/test_system_nidigital.py

```diff
@@ -558,6 +558,32 @@ def test_ppmu_source(multi_instrument_session):
     multi_instrument_session.pins['site0/LO0', 'site1/HI0'].ppmu_source()
 
 
+def test_configure_voltage_levels(multi_instrument_session):
+    assert round(multi_instrument_session.vil) == 0
+    assert round(multi_instrument_session.vih) == 3
+    assert round(multi_instrument_session.vol) == 2
+    assert round(multi_instrument_session.voh) == 2
+    assert round(multi_instrument_session.vterm) == 2
+    multi_instrument_session.load_pin_map(os.path.join(test_files_base_dir, "pin_map.pinmap"))
+    multi_instrument_session.pins['site0/PinA', 'site1/PinC'].configure_voltage_levels(1, 2, 3, 4, 5)
```

Comment:
Use keyword arguments like `(vil=1, vih=2..`. Improves code readability a lot.

I've tried to use keyword arguments for all API calls, let's try and keep it that way.

Reply:
Updated.